news/2018-07-01-osu-idol-2018-auditions-now-open.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "osu!idol 2018 Auditions Now Open"
+date: 2018-07-01 00:00:00 +0000
+---
+
+May I interest you in some singing? Want to compete with others to become the next osu!idol? Then grab a mic and join in, it’s that time of year again to submit your entries and show us what you've got!
+
+![](/wiki/shared/news/2018-07-01-osu-idol-2018-auditions-now-open/banner.jpg)
+
+### Hold on, what is this?
+
+The annual community driven singing contest - osu!idol! It's where contestants as a solo or a duet sing their way through multiple stages of different genres with the aim to reach the very top of the idol podium! The panel of judges we have prepared will evaluate your performances and ensure the best is standing at the final stretch. After that, you, the community, will decide with a voting to determine our next osu!idol! It's go time! 
```

Comment:
"decide to determine" is heavy and doesn't ready well. Reword this with something like "After that, you, the community, can vote to determine who our next osu!idol is"

Reply:
Changed, but went with "who our next *osu!idol will be*"